modules/mcuboot/Kconfig

```
@@ -49,6 +49,40 @@ config MCUBOOT_IMAGE_VERSION
 	  with version 0.1.2+3 can be overwritten by an uploaded image with
 	  0.1.2+2, but not by one with 0.1.1+2.
 
+menuconfig MCUBOOT_HARDWARE_DOWNGRADE_PREVENTION
+	bool "Downgrade prevention using hardware security counters"
+	depends on SOC_NRF5340_CPUAPP || SOC_NRF9160
+	help
+	  This option can be enabled by the application and will ensure
+	  that the MCUBOOT_HW_DOWNGRADE_PREVENTION Kconfig option is
+	  enabled in the MCUBoot image.
+
+if MCUBOOT_HARDWARE_DOWNGRADE_PREVENTION
+
+config MCUBOOT_HW_DOWNGRADE_PREVENTION_COUNTER_SLOTS
+	int "Number of available hardware counter slots"
+	default 240
+	range 2 300
+	help
+	  When MCUBOOT_HW_DOWNGRADE_PREVENTION is enabled MCUBOOT will use
+	  one hardware counter for each updateable image (UPDATEABLE_IMAGE_NUMBER).
+	  This configuration specifies how many counter slots will be allocated
+	  for each hardware counter. The hardware counters are stored in OTP storage.
+	  Rationale for the default number (240): Assume one update a month for
+	  10 years, then double that value just in case. This default fits
+	  comfortably within the "OTP" region of UICR.
```

Just added a `,` and `the`
```suggestion
	  When MCUBOOT_HW_DOWNGRADE_PREVENTION is enabled, MCUBOOT will use
	  one hardware counter for each updateable image (UPDATEABLE_IMAGE_NUMBER).
	  This configuration specifies how many counter slots will be allocated
	  for each hardware counter. The hardware counters are stored in OTP storage.
	  The rationale for the default number (240): Assume one update a month for
	  10 years, then double that value just in case. This default fits
	  comfortably within the "OTP" region of UICR.